src/main/java/za/ac/cput/school_management_grp33/service/location/impl/AddressServiceImp.java

```diff
@@ -0,0 +1,4 @@
+package za.ac.cput.school_management_grp33.service.location.impl;
+
+public class AddressServiceImp {
+}
```

Comment:
this belongs in the lookup package, not in the location. Also, it is `AddressServiceImpl`. You forgot the l at the end.